$NetBSD: patch-ac,v 1.1 2003/07/01 15:52:37 agc Exp $

Pass the value of ${PREFIX} down so that the help file can be found

--- makefile.bsd	2003/06/17 16:35:05	1.1
+++ makefile.bsd	2003/06/17 16:35:37
@@ -43,7 +43,7 @@
 SYSTEM	= -Dunix
 
 # select operating system environment
-DEFS	= -Dunix
+DEFS	= -Dunix -DPREFIX='"${PREFIX}"'
 
 
 # collection of system parameters for compilation
